File: 561.cpp
#include<algorithm>
class Solution {
public:
    int arrayPairSum(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int sum = 0;
        for(vector<int>::iterator it = nums.begin();it!=nums.end();it+=2)
            sum += *it;
        return sum;
        
    }
};

File: Python/561.py
class Solution(object):
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        print(nums)
        nums = nums[::2]
        print(nums)
        return sum(nums)
res = Solution()
nums=[1,3,2,4,5,6, 7,8,9,10]
print (res.arrayPairSum(nums))